core/amber/src/main/python/core/architecture/handlers/initialize_operator_logic_handler.py

```diff
@@ -12,5 +12,6 @@ def __call__(self, context: Context, command: cmd, *args, **kwargs):
         operator: type(Operator) = load_operator(command.code)
         context.dp._operator = operator()
         context.dp._operator.is_source = command.is_source
+        context.dp._operator.output_attribute_names = command.output_schema_fields
```

Comment:
the schema does not contain types?

Reply:
for now it doesn't.

Reply:
lets' make sure to include types in future PRs.